app/assets/stylesheets/header.scss

```diff
@@ -1,429 +1,333 @@
+        &.unread {
+          background-color: $background-grey;
+          color: $black;
+          .unread-toggle {
+            .entypo {
+              color: $black;
+            }
+          }
+        }
       }
     }
-  }
-
-  ul#user_menu {
-    overflow: hidden;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    text-shadow: 0 1px 0 #000;
-    color: rgb(147,147,147);
-    min-width: 100px;
-    cursor: pointer;
-    z-index: 10;
-    display: inline;
-    visibility: hidden;
-    top: -4px;
-    float: right;
-    margin: -2px -3px 0px 0px;
-    padding: 0;
-
-    > li {
-      display: block;
-      border-left: 1px solid #333;
-      border-right: 1px solid #333;
 
-      &:first-child {
-        visibility: visible;
-      }
-
-      &:last-child {
-        border-bottom: 1px solid #333;
-      }
+    #mark_all_read_link, #view_all_notifications {
+      padding: 10px 15px;
+      text-align: center;
+      font-weight: bold;
     }
 
-    &.active {
-      box-shadow: 0 0px 13px rgba(20,20,20,0.5);
-      background-color: rgb(34,30,30);
-      visibility: visible;
-
-      > li {
-        border-color: $border-dark-grey;
-      }
+    #view_all_notifications {
+      background: linear-gradient(to bottom, $blue, $dark-blue);
+      color: $white;
+      border: none;
     }
-  }
 
-  .user-menu-item a {
-    padding: 4px 8px 4px 40px;
-    height: 100%;
-    color: $text-grey;
-
-    &:hover {
-      background-color: $blue;
-      color: $highlight-white;
-      text-decoration: none;
+    #mark_all_read_link.disabled {
+      color: $link-disabled-grey;
+      &:hover, &:active, &:focus {
+        background: none;
+        cursor: default;
+      }
     }
   }
 
-  .user-menu-trigger {
-    padding: 10px 25px 10px 40px;
+  // Overridden rules for header dropdowns
+  div#user-menu{
+    border-left: 1px solid transparent;
+    border-right: 1px solid transparent;
 
-    &:hover {
-      color: $highlight-white;
-      .user-name { color: $highlight-white; }
+    &.open{
+      border-left: 1px solid $font_color;
+      border-right: 1px solid $font_color;
     }
-  }
 
-  .user-name {
-
-    &:hover {
-      background-color: transparent;
-      text-decoration: none;
+    ul > li > a{
+      padding-left: $img_height + 26px;
     }
-
-    &:focus { outline: none; }
   }
 
-  .user-menu-more-indicator {
-    position: absolute;
-    right: 5px;
-  }
-
-  .user-menu-avatar {
-    height: 25px;
-    width: 25px;
-    position: absolute;
-    left: 8px;
-    top: 8px;
-    display: block;
-  }
+  #global_search {
+    float: right;
+    position: relative;
+    margin: 0;
 
-  .header-nav {
-    font-weight: bold;
-    float: left;
-    height: 100%;
-    margin-right: 5px;
-    margin-top: 2px;
+    form {
+      padding: 0 6px;
 
-    a {
-      padding: 0 10px;
-      width: 100%;
-      &:hover { text-decoration: none; }
-    }
+      input {
+        @include transition(width);
+        background-color: #444;
```

Comment:
Color literals like `#444` should only be used in variable declarations; they should be referred to via variable everywhere else.